lib/constants/constants.dart

```diff
@@ -1,34 +1,52 @@
+import 'dart:ui';
+
 class Constants {
-  // Poster and Backdrop
-  static const posterLocation ="assets/images/poster.jpg";
-  static const backdropLocation = "assets/images/back.jpg";
-  static const posterYAligment = 20.0;
-  static const posterXAligment = 15.0;
-  static const posterHeight = 190.0;
-  static const posterWidth = 140.0;
-  static const posterStroke = 1.0;
+  // App colors
+  static const Color c1 = Color(0xFF001D3D);
+  static const Color c2 = Color(0xFF003566);
+
+  // Movie List
+  static const baseUrl = "https://image.tmdb.org/t/p/w500/";
+  static const posterHeight = 500.0;
+  static const posterPadding = 25.0;
+  static const posterRadius = 25.0;
+  static const cardMargin = 30.0;
+  static const cardRadiusTop = 25.0;
+  static const cardRadiusBottom = 10.0;
+
+  // Movie Detail
+  static const posterDetailYAligment = 20.0;
+  static const posterDetailXAligment = 10.0;
+  static const posterDetailHeight = 200.0;
+  static const posterDetailWidth = 140.0;
+  static const posterDetailStroke = 5.0;
+  static const originalTitleSize = 18.0;
+  static const likesSpacing = 30.0;
+  static const releaseDateSize = 18.0;
+  static const releaseDatePadding = 20.0;
+
+  // Stars Constants
+  static const two = 2;
```

Comment:
Why this name? Please choose a more suitable name

Reply:
Done.